platform/openide.explorer/src/org/openide/explorer/view/TableQuickSearchSupport.java

```diff
@@ -152,14 +154,18 @@ public void quickSearchConfirmed() {
         quickSearchInitialColumn = -1;
     }
 
+    private static final boolean RESTORE_PRIOR_SELECTION_AFTER_QUICK_SEARCH = false;
```

Comment:
What is the idea here? Is this a fancy way of commenting out code?

Reply:
Oh, sorry I left that in. I'll remove it...

(Yes, fancy way to comment out code, especially when I need to experiment with different UI behaviors for a while before deciding what works or not. It also survives later refactoring.)